source/util/migration_handler.ts

```diff
@@ -1,12 +1,7 @@
+import { Pool } from 'pg'
 import { migrate } from 'postgres-migrations'
 
 export async function applyMigrations(config: any): Promise<void> {
-	const dbConfig = {
-		database: config['database'] || 'postgres',
-		user: config['user'] || 'postgres',
-		password: config['password'] || 'postgres',
-		host: config['host'] || 'localhost',
-		port: config['port'] || 5432,
-	}
-	await migrate(dbConfig, __dirname + '/migrations')
+	const pool = new Pool(config)
```

Comment:
Oh, wait, should it be a `Client` rather than a `Pool`?

Reply:
Oh yeah, one off connections should probably use a `Client` instead.

Reply:
@nfriedly updated!